jsonschema-validation.xml

```diff
@@ -993,6 +995,116 @@
             </section>
         </section>
 
+        <section title='String-encoding non-JSON data'>
+
+            <section title="Foreword">
+                <t>
+                    Properties defined in this section indicate that an instance contains
+                    non-JSON data encoded in a JSON string.
+                    They describe the type of content and how it is encoded.
+                </t>
+                <t>
+                    These properties provide additional information required to interpret JSON data
+                    as rich multimedia documents.
+                </t>
+            </section>
+
+            <section title="Implementation requirements">
+                <t>
+                    Implementations MAY support keywords defined in this section.
+                    Should they choose to do so, they SHOULD offer an option to disable validation
```

Comment:
What is the reason / use case for this?

Reply:
This is to mirror what's stated for "format" keyword (in "Implementation requirements" section). The reasoning is that "format" and "media*" keywords are semantically similar, so implementations should treat them similarly.